File: bioinfo_training/lecture12_Align/02CallSNP/filter.pl
use strict;
use warnings;
my $file=shift;

open(F,"zcat $file |")||die "$file not exist\n";
open(O,'>',"$file.filter");
while(my $line=<F>){
    chomp;
    if($line=~m/^#/){
        print O "$line\n";
    }else{
        my @line=split(/\s+/,$line);
        if(%line[5]>=30){
        print O "$line\n";}
    }
}

File: bioinfo_training/population genetics/lecture12_Align/02CallSNP/filter.py
import re 
f = open('zcat 02.vcf/scaffold37_cov106.vcf.gz |')
lines = f.readlines()
for line in lines:
    if re.match('^\#(.*)', line):
        print(line)
    else:
        line.split()
        print(line)
